magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/lstchain_version.py

```diff
@@ -58,8 +64,19 @@ def version_lstchain(df_LST):
                 max_version = lstchain_versions[j]
 
         if max_version is None:
-            raise ValueError("issue with lstchain versions")
-        name = f"/fefs/aswg/data/real/DL1/{date}/{max_version}/tailcut84/dl1_LST-1.Run{run}.h5"
+            print(f"issue with lstchain versions for run {run}\n\n\n")
```

Comment:
you could add here that available versions: {version}, allowed versions: {lstchain_versions}